fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -232,10 +232,16 @@ class HttpClient(
         val contentLength = body.length
         if (contentLength != null && contentLength != -1L) {
             // The content has a known length, so no need to chunk
-            connection.setFixedLengthStreamingMode(contentLength.toLong())
+            with(connection) {
+            	addRequestProperty(Headers.CONTENT_LENGTH, contentLength.toString())
+            	setFixedLengthStreamingMode(contentLength.toInt())
+            }
         } else {
             // The content doesn't have a known length, so turn it into chunked
-            connection.setChunkedStreamingMode(4096)
```

Comment:
This is supposed to set it to chunked.

Reply:
Same as Fixed Length, it doesn't set the headers for it.

Reply:
If you're certain (check in the server; mock api, not in the headers object before it's sent), then this is fine with me.